test/still/preprocessor/image_test.exs

```diff
@@ -124,7 +124,7 @@ defmodule Still.Preprocessor.ImageTest do
           metadata: %{
             image_opts: %{
               sizes: [100, 200],
-              transformations: [color_filter: "grayscale_bt709"]
```

Comment:
Is this removal intended? Should we replace it with another one?

Reply:
It is because otherwise, the tests fail when changing between imageflow and imagemagick.